activitysim/core/chunk.py

```diff
@@ -1232,12 +1232,19 @@ def adaptive_chunked_choosers(
 
     chunk_tag = chunk_tag or trace_label
 
+    num_choosers = len(choosers.index)
+
+    explicit_and_odd_num_choosers = False
```

Comment:
The value of `explicit_and_odd_num_choosers` is not needed.  It's perfectly fine for some multiple of `rows_per_chunk` to overrun the end of the choosers by a bit, slicing beyond the end of the range.  If it were needed, checking for odd wouldn't be enough, we'd need to adjust based on the inverse of the number of chunks (e.g. 0.25 won't line up unless the total is divisible by 4 not 2)

Reply:
Please take a look at the changes in this commit: https://github.com/dhensle/activitysim/commit/1ed94c5f1169a2deb52e8673dc3ba169f431ea8a

I was hitting the assert statement for the alts, (not the choosers) which prompted me to try to adjust the `rows_per_chunk`.  But good point about odd not being good enough.  I removed this functionality and replace the assert statement with a simple check on overflow of the alt index. I think the solution in the above commit fixes the issue (it runs successfully), but suggest you take a look. Thanks!